introduction.md

```diff
@@ -9,3 +9,11 @@ Javascript
 HTML&CSS
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?